app/scripts/pages/config.js

```diff
@@ -9,6 +9,8 @@ var Config = React.createClass({
   getInitialState: function () {
     var t = this
     t.props.ipfs.config.show(function (err, configStream) {
+      if (err) console.log(err)
```

Comment:
This is mostly to make the linter happy tbh. There is no real error handling anywhere atm. 

Bows looks nice but what's really important is displaying errors to users in a friendly format, which we need to tackle soon